model/src/main/java/de/danoeh/antennapod/model/feed/PodcastIndexChapter.java

```diff
@@ -0,0 +1,29 @@
+package de.danoeh.antennapod.model.feed;
+
+public class PodcastIndexChapter extends Chapter {
+    public static final int PODCASTINDEX_CHAPTER = 4;
+
+    public PodcastIndexChapter(int startTime,
+                               String title,
+                               String link,
+                               String imageUrl) {
+        super(startTime * 1000, title, link, imageUrl);
+    }
+
+    @Override
+    public int getChapterType() {
+        return PODCASTINDEX_CHAPTER;
+    }
+
+    @Override
+    public String toString() {
+        return "PodcastIndexChapter "
+                + getTitle()
+                + " "
+                + getStart()
+                + " "
+                + getLink()
+                + " "
+                + getImageUrl();
```

Comment:
That uses a lot of vertical screen space

```suggestion
                + getTitle()
                + " " + getStart()
                + " " + getLink()
                + " " + getImageUrl();
```